ladybug_geometry/polyskel.py

```diff
@@ -0,0 +1,558 @@
+# coding=utf-8
+"""
+Implementation of the straight skeleton algorithm as described by Felkel and Obdržálek in their 1998 conference paper Straight skeleton implementation.
+"""
+
+from __future__ import division
+
+import logging
+import heapq
+from itertools import *
```

Comment:
The gods of PEP8 are very angry right now. 😄

Either import the module and reference it in the code or import only what you need to use here.

Reply:
FYI this is a Bottfy practice not mine :). I'll fix it.

Reply:
Good to know. Thanks for fixing it! 👍